utils.ts

```diff
@@ -274,6 +274,19 @@ export async function runInRepo(options: RunOptions & RepoOptions) {
 		} else {
 			overrides.vite = options.release
 		}
+		const viteManifest = JSON.parse(
+			await $`pnpm dlx pacote manifest ${options.release} --json`,
```

Comment:
@dominikg pacote is what npm info uses, so i added a call for it here to get the manifest of the release.

Reply:
why dlx instead of adding it to the repo as dev dependency and calling manifest directly? But i thought you said that pkg.pr.new didn't support the info service?

Reply:
> why dlx instead of adding it to the repo as dev dependency and calling manifest directly?

I can do it if you want! let me send a commit. 

> But i thought you said that pkg.pr.new didn't support the info service?

pacote does not rely on any service, it supports standalone tarballs and also packages from the registry.

Reply:
this would allow adding the override without running install first at the expense of downloading the full tarball just to inspect its package.json :see_no_evil: . I was under the impression that `npm info` had a more elegant way to get to that information.

Reply:
and yes, please don't use dlx, we want control over the version of pacote that is used and its more efficient to take it from local.

Reply:
> I was under the impression that npm info had a more elegant way to get to that information.

Yea, that'd be costly for servers if they want to consume the tarball themselves and parse the json! good for small packages, but for heavy packages, that's too much to unpack a tarball in the memory. Specially if that's cloudflare workers, which is what pkg.pr.new uses :)

> we want control over the version of pacote that is used and its more efficient to take it from local.

Noted, in the last commit.

Reply:
> but for heavy packages, that's too much to unpack a tarball in the memory.

I think you can process it in stream and only extract the `package.json` so that you won't need to keep the whole unpacked tarball. It's a bit hard though.